client/scala/scala-armada-client/src/test/scala/ArmadaClientSuite.scala

```diff
@@ -213,6 +248,79 @@ class ArmadaClientSuite extends munit.FunSuite {
     ac.deleteQueue(qName)
   }
 
+  test("ArmadaClient.cancelJobs()") {
+    val ac = ArmadaClient("localhost", testPort)
+    val qName = "nonexistent-queue-" + Random.alphanumeric.take(8).mkString
```

Comment:
why "nonexistent-queue"?

Reply:
From my editor's paste buffer, originally from a queue-check test!  Changed to just "queue-" prefix in commit ffdc1e18.